dali/pipeline/operators/reader/nvdecoder/cuvideodecoder.cc

```diff
@@ -175,6 +232,8 @@ int CUVideoDecoder::initialize(CUVIDEOFORMAT* format) {
     decoder_info_.DeinterlaceMode = cudaVideoDeinterlaceMode_Adaptive;
     decoder_info_.ulTargetWidth = format->display_area.right - format->display_area.left;
     decoder_info_.ulTargetHeight = format->display_area.bottom - format->display_area.top;
+    decoder_info_.ulMaxWidth = static_cast<unsigned long>(max_width_);  // NOLINT
```

Comment:
out of curiosity: why do you need `NOLINT` here?

Reply:
Linter recommends : `Use int16/int64/etc, rather than the C type long`
ulMaxWidth/ulMaxWidth is `unsigned long` in the API.